analytics-webapps/src/main/webapp/WEB-INF/jsp/analytics-table.jsp

```diff
@@ -17,7 +19,9 @@
 <portlet:actionURL var="saveSettingsURL" />
 
 <div class="VuetifyApp">
-    <% int generatedId = (int) (Math.random() * 1000000l); %>
+    <% int generatedId = (int) (Math.random() * 1000000l);
+    String limit = PropertyManager.getProperty("exo.analytics.table.default.limit");
```

Comment:
we should add a property in `AnalyticsTableFilter` and a new Setting value in Table settings UI